test/cpp_extensions/ninja_test/setup.py

```diff
@@ -0,0 +1,33 @@
+import sys
```

Comment:
This file was copied from `test/cpp_extensions/setup.py`, with one change: BuildExtension uses `use_ninja=True`.

Reply:
So for example, this file would read out the setting of `use_ninja` from environment variable